solidity/dashboard/src/utils/arithmetics.utils.js

```diff
@@ -25,3 +30,12 @@ export const lt = (a, b) => {
 export const lte = (a, b) => {
   return web3Utils.toBN(a).lte(web3Utils.toBN(b))
 }
+
+export const isZero = (a) => web3Utils.toBN(a).isZero()
+
+export const percentageOf = (value, total) => {
+  return web3Utils
+    .toBN(value)
+    .mul(ONE_HUNDRED)
+    .div(isZero(total) ? ONE : total)
```

Comment:
if `total` is zero, then the percentageOf will return `value * 100` which is not the correct result, right?
Maybe we shouldn't calculate `percentOf` in case of wrong values being passed to the function and return an error instead?